be.telenet.user.QSSTV.yml

```diff
@@ -0,0 +1,71 @@
+app-id: be.telenet.user.QSSTV
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: qsstv
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
+cleanup:
+  - /man
+  - /include
+  - /share/man
+  - /share/doc
+  - /doc
+  - '*.a'
+  - '*.la'
+  - /bin/usr
+modules:
+  - name: hamlib
+    sources:
+      - type: archive
+        url: https://github.com/Hamlib/Hamlib/releases/download/3.3/hamlib-3.3.tar.gz
+        sha256: c90b53949c767f049733b442cd6e0a48648b55d99d4df5ef3f852d985f45e880
+
+  - name: fftw3
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -Wno-dev
+      - -DENABLE_THREADS=ON
+    sources:
+      - type: archive
+        url: http://www.fftw.org/fftw-3.3.9.tar.gz
+        sha256: bf2c7ce40b04ae811af714deb512510cc2c17b9ab9d6ddcf49fe4487eea7af3d
+
+  - name: fftw3f
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -Wno-dev
+      - -DENABLE_THREADS=ON
+      - -DENABLE_FLOAT=ON
+    sources:
+      - type: archive
+        url: http://www.fftw.org/fftw-3.3.9.tar.gz
+        sha256: bf2c7ce40b04ae811af714deb512510cc2c17b9ab9d6ddcf49fe4487eea7af3d
+
+  - name: qsstv
+    buildsystem: simple
+    build-commands:
+      - mkdir -p $FLATPAK_DEST/share/applications/
+      - mkdir -p $FLATPAK_DEST/share/icons/hicolor/128x128/apps/
+      - cp $FLATPAK_BUILDER_BUILDDIR/qsstv/icons/qsstv.png $FLATPAK_DEST/share/icons/hicolor/128x128/apps/be.telenet.user.QSSTV.png
```

Comment:
The icon directory is probably incorrect. The actual icon size seems to be 42x42. I will fix this.

Reply:
Just this one left and it's ready to go, unless you can find a bigger icon in general.

Reply:
Is `$FLATPAK_DEST/share/icons/hicolor/42x42/apps/` a valid location supported by Flatpak? I am not sure.

Reply:
48x48 if anything, I doubt 42x42 would work.